src/renderer/components/virtual-list/virtual-list.tsx

```diff
@@ -87,7 +87,7 @@ function VirtualListInner<T extends { getId(): string } | string>({
   useEffect(() => {
     try {
       if (prevItems.current.length !== items.length || !isEqual(prevRowHeights.current, rowHeights)) {
-        listRef.current?.resetAfterIndex(0, false);
+        listRef.current?.resetAfterIndex(0);
```

Comment:
Removing force render delay attribute allows to update table properly after changing sorting order.